proof/crefine/RISCV64/StateRelation_C.thy

```diff
@@ -207,7 +207,7 @@ definition
   \<and> tcb_queue_relation getNext getPrev hp queue NULL qhead"
 
 fun
-  register_from_H :: "register \<Rightarrow> machine_word"
+  register_from_H :: "register \<Rightarrow> 8 word"
```

Comment:
It'd be good to give the `8 word` a name here, to avoid magic numbers.

Reply:
How about
```
type_synonym register_index = "8 word"
```
?